app/assets/javascripts/components/locales/fr.jsx

```diff
@@ -88,6 +89,7 @@ const fr = {
   "privacy.direct.short": "Direct",
   "privacy.direct.long": "N’afficher que pour les personnes mentionné⋅e⋅s",
   "privacy.change": "Ajuster la confidentialité du message",
+  "empty_column.notifications": "Vous n'avez pas de notifications. Interagissez avec d'autres personnes pour commencer la conversation.",
```

Comment:
This one is already on its way here :  https://github.com/tootsuite/mastodon/pull/1111